docker-for-mac/troubleshoot.md

```diff
@@ -27,73 +27,66 @@ the log data.
 Choose ![whale menu](images/whale-x.png){: .inline} -->
 **Diagnose & Feedback** from the menu bar.
 
-![Diagnose & Feedback](images/diagnose-feedback.png)
+![Diagnose & Feedback](images/diagnose-feedback.png){:width="600px"}
```

Comment:
The CSS for the entire site should make images responsive. Is there anything we can improve on the site so that you don't have to do this?

Reply:
Actually I was just doing something I saw Gwen do several times (for instance https://github.com/docker/docker.github.io/commit/fd753e5b275#diff-f0f81f92bf2604019320309498f91445R129 and below), so I thought it would help you guys.  And I was expecting this to be only a visual change, not anything related to bandwidth (or maybe I misunderstood what you meant).

Reply:
Yes, this is only a visual change. This syntax will generate the following html:
```
<img src="images/diagnose-feedback.png"  alt="Diagnose & Feedback" width="600px">
```
With the CSS we currently use in the docs site, you'll get this behavior:
* The maximum width for the image is 600px
* If the browser window is smaller than that, scale down the image

The images in these articles look great, so my question was about whether there's any change we could make to the CSS to make this the default behavior, so that you wouldn't have to add `{: width="600px"}` for most of the images.

Unfortunatelly I looked a bit more into this and don't think we could have better CSS styles. I've tried a couple of alternatives, but all of them have side-effects that I don't really like.